src/sixLesson/hashMap/Passport.java

```diff
@@ -0,0 +1,21 @@
+package sixLesson.hashMap;
+
+
+
+
+
+public class Passport {
+
+    private String pass;
```

Comment:
Should be two variables "Passport with series and number." but not single 'pass'.